### DISCUSSION TITLE
Discord invite has expired, please update.

### DISCUSSION THREAD

**Discussion by mzs114** [WGDashboard/WGDashboard#486] [General]
Hi @DaanSelen  I tried joining the discord invite regarding the PR #362 , but it seems to be expired, please create a new one.

Btw, if possible, please switch to something FLOSS/Libre, like Matrix (Unsure how the chat history is stored). Just a suggestion.

**Comment by DaanSelen** [comment]
Good Idea, I'll see if I can setup a Matrix room today

**Reply by DaanSelen** [reply]
https://matrix.to/#/#wgd:matrix.org

**Reply by DaanSelen** [reply]
Its really small right now, but we can discuss things there. If we need to discuss with Donald, I can always contact him.
